docs/defining-blocks.md

```diff
@@ -285,12 +292,14 @@ enum AlarmSound {
 namespace alarms {
     /**
      * Play an alarm sound for some time
-     * @param sound of the alarm to play, eg: AlarmSound.Annoy
-     * @param duration of the alarm sound, eg: 2000
+     * @param sound of the alarm to play
+     * @param duration of the alarm sound
      */
-    //% block="play an alarm sound || of %sound for %duration ms"
+    //% block="play an alarm sound || of $sound for $duration ms"
     //% duration.shadow=timePicker
     //% expandableArgumentMode="toggle"
+    //% sound.defl=AlarmSound.Annoy
```

Comment:
you might need to put quotes around `AlarmSound.Annoy` here... I'm not actually sure. The `.` might confuse our parser

Reply:
I checked it in the playground and it works fine.